src/global/Components/QrScanner/QrScanner.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useCallback } from 'react';
+import styled from 'styled-components';
+import Webcam from 'react-webcam';
+import { QRCode } from 'jsqr';
+import getPixels from 'get-pixels';
+/* eslint-disable */
+//@ts-ignore
+import Worker from 'worker-loader!./decodeWorker.worker';
+import VideoOverlay from './VideoOverlay';
+
+const Wrapper = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const Button = styled.button`
+  color: white;
+  opacity: 0.5;
+  width: 50px;
+  height: 50px;
+  border-radius: 50%;
+  display: flex;
+  position: absolute;
+  bottom: 30px;
+  left: 50%;
+  transform: translate(-50%);
+`;
+
+type ExtendedWebcam = {
+  ref: React.RefObject<Webcam>;
```

Comment:
You can add RefObject into import at the top of this file and use just RefObject insted of React.RefObject

Reply:
done